junction/templates/proposals/partials/proposal-list--review-items.html

```diff
@@ -47,10 +47,18 @@ <h3 class="proposal--title">
                                 <a class='tag tag__proposal_type label label-proposal-type inline-block'>{{ proposal.proposal_type }}</a>
                                 <a class='tag tag__proposal_section label label-proposal-type inline-block'>{{ proposal.proposal_section }}</a>
                                 {% if proposal|reviewer_comments:request.user %}
-                                    <a class='tag tag__proposal_section label label-proposal-type inline-block'  data-toggle="tooltip" data-placement="bottom" data-original-title="Left a review comment">
+                                <a class='tag tag__proposal_section label label-proposal-type inline-block'  data-toggle="tooltip" data-placement="bottom" data-original-title="Left a review comment">
                                         <i class="fa fa-comments-o"></i>
                                     </a>
                                 {% endif %}
+                                    {% if proposal|has_content_urls %}
```

Comment:
Since the template tag will return all urls. It will be idle to replace with for loop and can have else part.